src/moveFile.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const moveFile = (oldPath, newPath) => {
+  const destPath = fs.existsSync(newPath)
+    ? path.join(newPath, path.basename(oldPath))
+    : newPath;
+
+  const isDestDir = destPath.endsWith('/');
+
+  const destDir = isDestDir ? destPath.slice(0, -1) : path.dirname(destPath);
+
+  if (!fs.existsSync(destDir)) {
+    throw new Error('The destination directory does not exist.');
+  }
+
+  if (isDestDir) {
+    fs.copyFileSync(oldPath, destPath);
```

Comment:
fs.copyFileSync is used to make a copy of the file in the new path, but the task requires moving the file, not copying. fs.renameSync should be used instead.